.github/actions/test-dotnet/replace-xunit-maxParallelThreads.ps1

```diff
@@ -0,0 +1,12 @@
+param ($MaxParallelThreads)
+
+Write-Output "Replacing maxParallelThreads in xunit.runner.json files to $MaxParallelThreads."
+
+$counter = 0
+
+Get-ChildItem "*/**/xunit.runner.json" -Recurse | ForEach {
+     $counter++
+     (Get-Content $_) | ForEach  {$_ -Replace '"maxParallelThreads":\s*([-\d]*)', "`"maxParallelThreads`": $MaxParallelThreads"} | Set-Content $_
```

Comment:
This line is too long. You can add line breaks after the pipe (other binary operators work too) to format the code nicely without having to escape anything.

Reply:
It was 145 characters with `ForEach`. I thought we are doing 120 character limit.

Reply:
> For legibility try to limit lines of code to 120 characters (soft limit) or 150 at most (hard limit, code analyzers will warn about it too), if breaking at 120 would hurt compactness or legibility.